pkg/chains/chains.go

```diff
@@ -129,99 +197,96 @@ func BtcDustOffset() int64 {
 // DefaultChainsList returns a list of default chains
 func DefaultChainsList() []*Chain {
 	return chainListPointers([]Chain{
```

Comment:
Can we consider removing the pointers here?
I think it is not necessary

Reply:
Yeah, I was thinking of doing that too, but it would mean a lot of changes on the codebase, making the PR difficult to review 
I will add the changes in a separate PR 
https://github.com/zeta-chain/node/issues/2080